.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ concurrency:
 jobs:
   golangci:
     name: lint
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-20.04
```

Comment:
Why do you want to downgrade the version of Ubuntu?

Reply:
Fotget, I think test the lint.